src/swift/Queue.swift

```diff
@@ -116,9 +116,17 @@ extension DispatchQueue {
 		_swift_dispatch_apply_current(iterations, work)
 	}
 
+    #if os(Android)
+	private static let androidSwiftMain = DispatchQueue(label: "AndroidSwiftMain", qos: .userInteractive)
+
+	public class var main: DispatchQueue {
+        return androidSwiftMain
```

Comment:
Maybe lets crash here?

Reply:
maybe. I just cherry-pick as it was before